### ISSUE TITLE
Problem with displaying clojure's sorted-set

### ISSUE THREAD

**Issue by malloc82** [vlaaad/reveal#26]
When I use ```vlaaad.reveal.nrepl/middleware```, I cannot print sorted-set anywhere. When an expression such as ```(apply sorted-set [1 2 3])``` is evaluation in cider repl, I get the following error message

```
user> (apply sorted-set [1 2 3])
Error printing return value (ClassCastException) at clojure.lang.Keyword/compareTo (Keyword.java:114).
class java.lang.Long cannot be cast to class clojure.lang.Keyword (java.lang.Long is in module java.base of loader 'bootstrap'; clojure.lang.Keyword is in unnamed module of loader 'app')
user> 
```

If I send the result to tap, I don't get any error in repl

```
user> (tap> (apply sorted-set [1 2 3]))
true
user> 
```

However, the value is not printed in reveal window, no error either. This is the only data structure I'm having problem with reveal so far.

**Comment by vlaaad** [comment]
Thanks for the report, I can reproduce the issue! Working on it...

**Comment by vlaaad** [comment]
Fixed in `1.3.195`, please update :)

**Comment by malloc82** [comment]
Bravo! 👍
